CHANGELOG.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 in development
 --------------
 
+Changed:
```

Comment:
Also, since changelog entries tend to contain present form and not past, maybe we should use same format for those section names as well - New features / Improvements / Additions / Fixes